app/indexers/work_indexer.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class WorkIndexer
+  def self.call(work, commit: false)
+    work.versions.published.map do |version|
```

Comment:
@rschenk I suppose we could exclude the latest published version here and that would avoid the duplicate record issue. There'd always be x number of records in Solr, 1 work record and x - 1 work version records.

Reply:
Would that somehow cause mayhem if we wanted to show the latest published version via a blacklight Catalog#show page?

Reply:
@rschenk maybe? We don't really know how the catalog is going to figure into all of this yet, so I'm inclined to just leave it alone. One thing that might cause some weirdness is that if you search for both works and work versions, the most recent version's result weight in the  relevance ranking might get skewed because there are two documents with the same metadata. I don't think this would happen when you're searching for just works _or_ versions. In either case, I don't think this is something to be concerned about now, since we'll only be searching for works until another use case comes up.

Reply:
Yah I agree, let's punt. For all we know, we could end up showing the Postgres version on that page